packages/vscode-extension/messages.json

```diff
@@ -21,9 +21,10 @@
 	"LIST_OBJECTS_LISTING": "Listing objects...",
 	"SEE_DETAILS": "See details",
 
-	"UPLOAD_FILE_UPLOADING": "Uploading files...",
+	"UPLOAD_FILE_ERROR_UPLOAD_FILE_FOLDER_RESTRICTION": "The file being uploaded should be in a supported folder inside the FileCabinet folder. For information about the supported folders, see https://system.netsuite.com/app/help/helpcenter.nl?fid=section_4724963992.html.",
```

Comment:
Could we change it to `The file to upload should be in a supported folder inside the FileCabinet folder. For more information...`?

Reply:
Updated.